.github/workflows/coderabbit-resolve-check.yml

```
@@ -0,0 +1,41 @@
+      - name: Setup GitHub CLI
+        uses: cli/cli@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check for unresolved CodeRabbit comments
+        run: |
+          set -euo pipefail
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          OWNER=${{ github.repository_owner }}
+          REPO=${{ github.event.repository.name }}
+        run: |
```

actionlint (1.7.7)

key "run" is duplicated in element of "steps" section. previously defined at line:15,col:9

(syntax-check)